src/DotLiquid.Tests/Tags/Html/TableRowTests.cs

```diff
@@ -45,5 +54,14 @@ public void TestHtmlOffsetLimit()
                 "{% tablerow n in numbers cols:2 offset:1 limit:3 %}{{n}}{% endtablerow %}",
                 Hash.FromAnonymousObject(new { numbers = new[] { 1, 2, 3, 4, 5, 6 } }));
         }
+
+        [Test]
+        public void TestDynamicVariableColsOffsetLimit()

```

Comment:
Should this test be separated into separate tests? A failure wouldn't indicate which dynamic is failing. Result can be the same but swap the dynamic variable.

Reply:
i follow existing tests. this test is following previous one

Reply:
This document explains what named parameters are, with examples: https://learn.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#named-arguments

Reply:
i update this test